pkgs/development/python-modules/mpi4py/default.nix

```diff
@@ -31,7 +32,7 @@ buildPythonPackage rec {
   ];
 
   dependencies = [
-    mpi
+    (toPythonModule mpi)
```

Comment:
```suggestion
    # Use toPythonModule so that also the mpi executables will be propagated to
    # generated Python environment.
    (toPythonModule mpi)
```